modules/security/pki/default.nix

```diff
@@ -9,13 +9,12 @@ let
     blacklist = cfg.caCertificateBlacklist;
   };
 
-  caCertificates = pkgs.runCommand "ca-certificates.crt"
-    { files =
-        cfg.certificateFiles ++
-        [ (builtins.toFile "extra.crt" (concatStringsSep "\n" cfg.certificates)) ];
-     }
+  caCertificates = pkgs.runCommand "ca-certificates.crt" {}
     ''
-      cat $files > $out
+      cat ${escapeShellArgs (
+        cfg.certificateFiles ++
```

Comment:
This was not able to be kept in the env vars section of `pkgs.runCommand`, because cat still sees it as a single argument